model/map/src/main/java/org/keycloak/models/map/user/UserConsentEntity.java

```diff
@@ -37,6 +37,7 @@
     private Long createdDate;
     private Long lastUpdatedDate;
     private boolean updated;
+    private String userConsentId;
```

Comment:
By recent keycloak's convention, it might be better not to add or modify JPA entity because it causes a schema change. For example, Client Policies/Profiles avoid adding entities by storing JSON string itself to realm's attribute (existing key-value store).

Reply:
will be remove and stored as described above